lib/fireblocks/api/client.go

```diff
@@ -0,0 +1,123 @@
+package fireblocks
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+
+	"github.com/golang-jwt/jwt"
+)
+
+type Client struct {
+	apiKey       string
+	clientSecret string
+	host         string
+	http         http.Client
+}
+
+const TransactionEndpoint string = "v1/transactions"
+
+func NewClient(apiKey string, clientSecret string, host string) *Client {
```

Comment:
not sure host should be provided, isn't that constant? Where does the value come from?

Reply:
different hosts for different environments
Mainnet/Testnet: https://api.fireblocks.io/v1
Developer Sandbox: https://sandbox-api.fireblocks.io/v1
Figured we would have these as an environment variable

Reply:
there is a different url for sandbox, vs testnet/mainnet